public/js/govuk.js

```diff
@@ -819,10 +819,13 @@ Accordion.prototype.initControls = function () {
   this.$openAllButton.setAttribute('type', 'button');
 
   // Create control wrapper and add controls to it
-  var accordionControls = document.createElement('div');
-  accordionControls.setAttribute('class', this.controlsClass);
+  // var accordionControls = document.createElement('div');
+  // accordionControls.setAttribute('class', this.controlsClass);
+  // accordionControls.appendChild(this.$openAllButton);
+  // this.$module.insertBefore(accordionControls, this.$module.firstChild);
```

Comment:
Do we need the comments?

Reply:
It's not our code, I'm gonna leave them there

Reply:
Ah, ok, I thought it might not be thank you!